webpages/settings/index.html

```diff
@@ -251,23 +251,29 @@ <h1 v-cloak>{{ msg("settings") }}</h1>
       </button>
 
       <!-- This is the main menu, where the searchbar and the addon items are located -->
-      <div class="addons-block">
+      <main class="addons-block">
         <div class="addons-block-header">
           <div v-if="!isIframe" class="category-header-title" v-cloak>
             <div v-if="relatedAddonsOpen" class="related-addons-header">
               <button class="arrow-button" :title="msg('back')" @click="backRelatedAddon()">
                 <img src="../../images/icons/left-arrow.svg" draggable="false" />
               </button>
-              <span>{{ msg("relatedTo", relatedToAddonName) }}</span>
+              <span role="heading" aria-level="1">{{ msg("relatedTo", relatedToAddonName) }}</span>
             </div>
-            <span v-else>{{ selectedCategoryName }}</span>
+            <span role="heading" aria-level="1" v-else>{{ selectedCategoryName }}</span>
           </div>
-          <div v-cloak class="search-box" v-if="!relatedAddonsOpen" :class="{smallMode: smallMode === true}">
+          <div
+            role="search"
+            v-cloak
+            class="search-box"
+            v-if="!relatedAddonsOpen"
+            :class="{smallMode: smallMode === true}"
+          >
```

Comment:
This `search` landmark is inside the `main` landmark, which is probably fine because it filters through the addon list, which is also part of the `main` landmark. However, since it doesn't perform a site-wide search, should it be labeled? "Search addons" and "filter addons" would be good as labels or as placeholder text.